### DISCUSSION TITLE
[Duplicate] zh-CN/double-fault-exceptions/

### DISCUSSION THREAD

**Discussion by giscus** [phil-opp/blog_os#1227] [Post Comments (translated)]
# Double Faults (zh-CN)

在这篇文章中，我们会探索 double fault 异常的细节，它的触发条件是调用错误处理函数失败。通过捕获该异常，我们可以阻止致命的 triple faults 异常导致系统重启。为了尽可能避免 triple faults ，我们会在一个独立的内核栈配置 中断栈表 来捕捉 double fault…

https://os.phil-opp.com/zh-CN/double-fault-exceptions/

**Comment by XYZboom** [comment]
my qemu send an error message 
```
panicked at 'misaligned pointer dereference: address must be a multiple of 0x8 but is 0xdeadbeef'
```
I tried 0xdeadbee0, the qemu looped into reboot successfully.
